content/blog/espressif-ide/espressif-ide-3-1-0-release/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "New Release: Espressif IDE 3.1.0"
+date: 2024-10-07
+disableComments : false
+tags: ["IDE", "ESP-IDF", "Eclipse", "Espressif IDE"]
+featureAsset: "img/featured/featured-espressif-ide.webp"
+showAuthor: false
+authors:
+    - pedro-minatel
+---
+
+## Espressif-IDE
+
+Today we announce the new version of the Espressif IDE, the version 3.1.0. This new version adds new functionalities and bug fixes.
+
+<figure style="width: 90%; margin: 0 auto; text-align: center;">
+    <img
+        src="./espressif-ide-new-target.webp"
+        alt="espressif-ide-new-target.webp"
+        title="espressif-ide-new-target.webp"
+        style="width: 100%;"
+    />
+    <figcaption>Espressif-IDE New target setup</figcaption>
+</figure>
+
+## New features
+
+- Updated Eclipse Platform Support
+  - Support for the Eclipse 2024-06 release, ensuring compatibility with the latest Eclipse features and enhancements.
+- CDT LSP 2.0 Integration
+  - Full support for CDT Language Server Protocol (LSP) 2.0, providing enhanced code navigation, syntax highlighting, and error checking.
```

Comment:
It would be great if we can link CDT-LSP 2.0 with the https://github.com/eclipse-cdt/cdt-lsp/releases/tag/CDT_LSP_2_0_0